src/main/java/org/codebusters/audiogeek/preferencesagent/domain/mygenres/exception/MyGenresDomainException.java

```diff
@@ -0,0 +1,34 @@
+package org.codebusters.audiogeek.preferencesagent.domain.mygenres.exception;
+
+import org.codebusters.audiogeek.preferencesagent.infrastructure.exception.ErrorData;
+import org.codebusters.audiogeek.preferencesagent.infrastructure.exception.ExceptionModel;
+import org.springframework.http.HttpStatus;
+
+import static java.lang.String.format;
+
+
+public class MyGenresDomainException extends RuntimeException implements ExceptionModel {
```

Comment:
- Why domain relate to infrastructure (ExceptionModel)
- Why domain exception have an HTTP error code? The domain should never know about the client (application).